tests/test_pe_reports.py

```diff
@@ -1,11 +1,61 @@
 """File to be used to create test for pe-reports."""
 
 # Standard Python Libraries
+import sys
 import unittest
+from unittest.mock import patch
+
+# Third-Party Libraries
+import pytest
+
+# cisagov Libraries
+import pe_reports.report_generator
+
+PROJECT_VERSION = pe_reports.__version__
 
 # TODO: Replace current dummy test with useful tests - https://github.com/cisagov/pe-reports/issues/3#issue-909531010
 
 
+def test_output(capsys):
+    """Routine to test captured output."""
+    print("hello")
+    sys.stderr.write("world\n")
+    captured = capsys.readouterr()
+    assert captured.out == "hello\n"
+    assert captured.err == "world\n"
+    print("next")
+    captured = capsys.readouterr()
+    assert captured.out == "next\n"
```

Comment:
This test doesn't test anything in this project- it can be deleted.  The same goes for the `TestingStringMethods` class below.

Reply:
Resolved in 124615b